acoular/demo/__init__.py

```diff
@@ -1,7 +1,7 @@
 # coding=UTF-8

```

Comment:
```suggestion
```

By default, Python source files are treated as encoded in UTF-8. (https://docs.python.org/3/tutorial/interpreter.html#source-code-encoding)

Reply:
I didn't add this. I suggest to "unignore" `UP012` and let ruff handle this.

Reply:
I just confused `UP009` with `UP012`. I will remove the encoding statements but will also unignore `UP012` while at it.